src/Game.cpp

```diff
@@ -215,6 +414,7 @@ int sub_401190() {
     sub_6134D0(gamePath);
     glo_6C3250.sub_53B6A0();
     sub_5ED7B0(glo_6C3250.get_unk44(), glo_6C3250.get_unk48(), glo_6C3250.get_unk4C(), 32, 0);
+    glo_windowed_mode = 1; // Added to force window mode whilst developing
```

Comment:
This looks like it was added back as part of the rebase. This should be dropped. There's a configuration for it now, so doing this just hurts the development builds.